app/Models/Location.php

```diff
@@ -8,4 +8,8 @@ class Location extends Model
 {
     public $incrementing = false;
     public $timestamps = false;
+
+    protected $fillable = [
+        'id', 'address', 'lat', 'lng', 'radius'
```

Comment:
I would think about renaming lng to long, just cause it might not be intuitive in the future. up to you though

Reply:
Doing so would require to rename the table column, as that is what I am using for reference